packages/react-instantsearch/src/components/RatingMenu.js

```diff
@@ -1,3 +1,4 @@
+import { find, fill } from 'lodash';
```

Comment:
probably best to depend on /find and /fill ?

Reply:
The `babel-rollup-plugin` handle that for us.

Reply:
didn't know we used it 🙌